### ISSUE TITLE
[Keyboard accessibility] Cannot expand Zotero menu and cannot focus its sub-menus in Google Docs

### ISSUE THREAD

**Issue by jooyoungseo** [zotero/zotero-connectors#423]
I am a screen reader who needs to manipulate everything via keyboard. Unlike other menus, Zotero menu in Google Docs is not expandable via keyboard arrow keys and I cannot focus on its sub-menus. This issue can be found in any browsers.

Please seriously consider fixing this issue for keyboard-only users.

**Comment by dstillman** [comment]
@jooyoungseo: This is fixed now in the <a href="https://www.zotero.org/support/beta_builds#zotero_connector_beta">Zotero Connector beta</a> for Firefox, and it will be included in the next versions of the Chrome, Edge, and Safari extensions. Thanks for reporting this.

**Comment by jooyoungseo** [comment]
@dstillman -- Thanks, but issue has not been resolved. I cannot expand the Zotero menu  item and cannot arrow down its sub-menus. Tested with Zotero beta Firefox on Windows 11:

Version
5.0.107beta1
Last Updated
February 25, 2023

**Comment by dstillman** [comment]
Can you say exactly what you're trying? E.g., in Firefox with 5.0.107beta1, I can press Option-Shift-Z to focus the Zotero menu (or Option-/ followed by left arrow), use the up/down arrow keys to move through the menu, and press Enter to select an option.

It looks like window focus isn't always working properly when using the keyboard, though — e.g., sometimes you have to Alt-Tab to a Zotero window to bring it to the front. We'll see if we can fix that.

**Comment by jooyoungseo** [comment]
@dstillman Sure thing. I am sharing the reproducible steps below:

1. On Windows, load [NVDA screen reader](https://www.nvaccess.org/download/).

1. Open any google Doc in Firefox.

1. Press Alt+Shift+Z to focus in the Zotero menu.

1. Press DownArrow.

I don't hear any focus change.

**Comment by dstillman** [comment]
OK, I can reproduce that.

**Comment by dstillman** [comment]
@jooyoungseo: OK, please try the latest Zotero Connector beta.
